src/metabase/email/messages.clj

```diff
@@ -392,7 +392,7 @@
   point in the future; for now, this function is a stopgap.
 
   Results are streamed synchronously. Caller is responsible for closing `os` when this call is complete."
-  [export-format format-rows? ^OutputStream os {{:keys [rows]} :data, database-id :database_id, :as results}]
+  [^OutputStream os {:keys [export-format format-rows? pivot?]} {{:keys [rows]} :data, database-id :database_id, :as results}]
```

Comment:
slightly refactor the args here to pass the flags in in a map instead.

Reply:
nice